admiral/cmd/admiral/cmd/root.go

```diff
@@ -85,6 +85,8 @@ func GetRootCmd(args []string) *cobra.Command {
 		"The workload identity  key, on deployment which holds identity value used to generate cname by admiral. Default label key will be \"identity\" Admiral will look for a label with this key. If present, that will be used. If not, it will try an annotation (for use cases where an identity is longer than 63 chars)")
 	rootCmd.PersistentFlags().StringVar(&params.LabelSet.GlobalTrafficDeploymentLabel, "globaltraffic_deployment_label", "identity",
 		"The label key which will be used to tie globaltrafficpolicy objects to deployments. Configured separately to the workload identity key because this one won't fall back to annotations.")
+	rootCmd.PersistentFlags().StringVar(&params.WorkloadSidecarUpdates, "workload_sidecar_updates", "disabled",
```

Comment:
Can we be more explicit about what the `disabled` state is? Would that just mean every sidecar has `*/*`? (And is there any configuration that needs to be done to make it be that way?)